### PR TITLE
simpson's paradox is added

### PR DESCRIPTION
**Pull request by 11chankun**

### CODE REVIEW

File: intents/simpson_paradox.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
+""" 
+	**Some improvement will be done in this code, all ideas for better implementation will be appreciated.
```
Comment (chhavip) [inline intents/simpson_paradox.py:2]: Restrict line width to 80 chars everywhere.

File: intents/simpson_paradox.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+	1. commenting part will be done later. 
+	2. Implementation part will be improved by using some python function.
+	3. Variable name will be changed according to the need.
+"""
+from Utilities import aspects
```
Comment (chhavip) [inline intents/simpson_paradox.py:7]: Leave a line after comments and before imports

File: intents/simpson_paradox.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+				negative_count = negative_count + 1;
+			elif difference > 0:
+				positive_count = positive_count + 1;
+			else:
+				zero_count = zero_count + 1;
+			total_count = total_count + 1;
+		row_i = row_i + 1
+	return (positive_count/total_count)*100
+    
+def simpson_paradox(table, dimension, grouping, summary_operation_column, slice_compare, summary_operator):
```
Comment (chhavip) [inline intents/simpson_paradox.py:73]: The function that will be called from another module (essentially be public) should be at the top, while private functions below. Also it is not clear form the name what this function does or returns, does it calculate simpson's paradox and returns a value or just prints a bunch of stuff?

File: intents/simpson_paradox.py [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+	**Some improvement will be done in this code, all ideas for better implementation will be appreciated.
+	1. commenting part will be done later. 
+	2. Implementation part will be improved by using some python function.
+	3. Variable name will be changed according to the need.
+"""
+from Utilities import aspects
+from Utilities import enumerations
+import pandas as pd
+
+def check_simpson_paradox(table, dimension, slice_compare, summary_operator):
```
Comment (chhavip) [inline intents/simpson_paradox.py:11]: Add some comments / documentation of what the function is trying to do.

**Comment by 11chankun** [comment]
I need to do many changes because I got to know many new features about it.

**Pull request closed without merging**